### PR TITLE
Add two tests for Graphics.DrawString.

### PR DESCRIPTION
**Pull request by mellinoe**
This adds two very simple tests for `Graphics.DrawString`.

@hughbe @KostaVlev

### CODE REVIEW

File: src/System.Drawing.Common/tests/GraphicsTests.cs [hunk 1]
```diff
@@ -3481,5 +3506,23 @@ private static void VerifyGraphics(Graphics graphics, RectangleF expectedVisible
             Assert.Equal(new Matrix(), graphics.Transform);
             Assert.Equal(expectedVisibleClipBounds, graphics.VisibleClipBounds);
         }
+
```
Comment (hughbe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Can we move this to Helpers,cs?

File: src/System.Drawing.Common/tests/GraphicsTests.cs [hunk 2]
```diff
@@ -3460,6 +3461,30 @@ public void Clear_Disposed_ThrowsArgumentException()
...
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawString_CompositingModeSourceCopy_ThrowsArgumentException()
+        {
+            using (var image = new Bitmap(10, 10))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                graphics.CompositingMode = CompositingMode.SourceCopy;
+                AssertExtensions.Throws<ArgumentException>(
+                    null,
+                    () => graphics.DrawString("Test text", SystemFonts.DefaultFont, Brushes.White, new Point()));
```
Comment (hughbe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Could use Point.Empty

File: src/System.Drawing.Common/tests/GraphicsTests.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: (Line below in review)

**Comment by mellinoe** [comment]
@hughbe I am writing an application that uses System.Drawing (as more of a "real-world" test case), and ran into this behavior. I wasn't actually sure of the behavior myself so I just wrote a couple of tests to figure out what is supposed to happen, and figured I might as well check them in, since we didn't have any tests for `DrawString` previously.

File: src/System.Drawing.Common/tests/GraphicsTests.cs (hunk 1 above)
Comment (mellinoe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Yeah, sounds good.

File: src/System.Drawing.Common/tests/GraphicsTests.cs (hunk 2 above)
Comment (mellinoe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Personally, I find the concept of an "empty point" a bit confusing... I think I prefer `new Point()`.

**Comment by mellinoe** [comment]
@dotnet-bot Test Linux x64 Release Build

The Drawing tests seg-faulted again -- I'm trying to debug through the core dump now to figure out why. It was on Ubuntu 16.04 so I should have an easier time accomplishing that.

**Pull request merged**